h/static/styles/thread.scss

```diff
@@ -48,15 +53,37 @@ $thread-padding: $annotation-card-left-padding;
 
   & > ul {
     padding-left: $thread-padding + 3px;
+    padding-right: $thread-padding + 3px;
+    padding-top: $thread-padding - 9px;
     margin-left: -$thread-padding;
+    margin-right: -$thread-padding;
+    margin-top: $thread-padding;
   }
 
   // nested threads for annotation replies
   .thread {
-    border-left: 1px dotted $gray-light;
+    border-left: 1px dotted $grey-3;
     padding: 0;
     padding-left: $thread-padding;
     margin-top: 10px;
+
+    &:hover {
+      & > .threadexp > span {
+        color: $grey-7;
+      }
+    }
+  }
+
+  .thread-message:hover {
+    & > div >.annotation-footer .annotation-action-btn,
```

Comment:
Can we reduce the specificity of the selectors? The more specific selectors get the harder it is to reason about what gets applied in different situations. Could this be simplified to:

```scss
.thread-message:hover {
  .annotation-action-btn, .annotation-replies__link, .annotation-replies__count {
    color: $grey-6;
  }
}
```

?